log-router/templates/daemonset.yaml

```diff
@@ -105,10 +105,22 @@ spec:
           - --label-selector={{- range $k, $v := .Values.labelSelector.matchLabels }}{{$k}}={{$v}},
           {{- end }}
           {{- end }}
+          {{- if .Values.enforceNamespaces }}
+          {{- if .Values.namespaces}}
           {{- range  .Values.namespaces }}
           - --namespaces
           - "{{ . }}"
           {{- end }}
+          {{- else }}
+          - --namespaces
+          - "{{ .Release.Namespace }}"          
+          {{- end }}
+          {{- else }}
+          {{- range  .Values.namespaces }}
+          - --namespaces
+          - "{{ . }}"
+          {{- end }}
+          {{- end }}
```

Comment:
I would suggest something like: 
```
{{- if .Values.useReleaseNamespaceOnly }}
- --namespaces
- "{{ .Release.Namespace }}" 
{{- else }}
{{- range  .Values.namespaces }}
- --namespaces
- "{{ . }}"
{{- end }}
{{- end }}
```

And add a comment to `values.yaml` explaining the flag better

Reply:
Updated!